lib/strategies/accept_header_strategy.rb

```diff
@@ -0,0 +1,42 @@
+module Phenotype
+  class AcceptHeaderStrategy
+    attr_reader :env, :content_type, :version_key
+
+    def initialize(app_name: 'rentpath',
+                   version_key: 'version', 
+                   format: 'json')
+      @version_key = version_key
+      @content_type = build_content_type(app_name, format)
+    end
+
+    def get_version(env)
```

Comment:
See https://github.com/rentpath/phenotype/pull/1 as to why we don't just pass env to the initializer.